curation_concerns-models/app/actors/curation_concerns/work_actor_behavior.rb

```diff
@@ -80,13 +80,4 @@ def attach_file(file)
     def visibility_attributes
       raw_attributes.slice(:visibility, :visibility_during_lease, :visibility_after_lease, :lease_expiration_date, :embargo_release_date, :visibility_during_embargo, :visibility_after_embargo)
     end
-
-    def valid_file?(file_path)
```

Comment:
I assume you deleted these because they pertain to files and this module is for works, so they're not called?